src/test/kotlin/no/nav/familie/tilbake/iverksettvedtak/TilbakekrevingsvedtakBeregningServiceTest.kt

```diff
@@ -941,6 +946,117 @@ internal class TilbakekrevingsvedtakBeregningServiceTest : OppslagSpringRunnerTe
         )
     }
 
+    @Test
+    fun `beregnVedtaksperioder skal beregne periode med 100 prosent tilbakekreving og renter skal rundes ned`() {
+        kravgrunnlagRepository.deleteById(kravgrunnlag.id)
+        fagsakRepository.update(fagsakRepository.findByIdOrThrow(fagsak.id).copy(fagsystem = Fagsystem.EF))
+
+        val kravgrunnlagxml = readXml("/kravgrunnlagxml/kravgrunnlag_EF_med_renter_avrundingsfeil_ned.xml")
+        val kravgrunnlag = KravgrunnlagMapper.tilKravgrunnlag431(
+            KravgrunnlagUtil.unmarshalKravgrunnlag(kravgrunnlagxml),
+            behandling.id
+        )
+        kravgrunnlagRepository.insert(kravgrunnlag)
+
+        val periode = kravgrunnlag.perioder.first().periode
+
+        val grovUaktsomhetPeriode = lagGrovtUaktsomVilkårsvurderingsperiode(periode.fom, periode.tom)
+
+        vilkårsvurderingService.lagreVilkårsvurdering(
+            behandling.id,
+            BehandlingsstegVilkårsvurderingDto(
+                listOf(grovUaktsomhetPeriode)
+            )
+        )
+
+        val tilbakekrevingsperioder = vedtakBeregningService.beregnVedtaksperioder(behandling.id, kravgrunnlag)
+            .sortedBy { it.periode.fom }
+        tilbakekrevingsperioder.size shouldBe 1
+        shouldNotThrowAny { iverksettelseService.validerBeløp(behandling.id, tilbakekrevingsperioder) }
+
+        val tilbakekrevingsperiode = tilbakekrevingsperioder[0]
+        tilbakekrevingsperiode.periode shouldBe periode
+        tilbakekrevingsperiode.renter shouldBe BigDecimal(1860)
+        tilbakekrevingsperiode.renter shouldNotBe BigDecimal(1861)
```

Comment:
Trenger man begge disse 👀

Reply:
Nei, men tok den med for å vise hva som ville bli hvis man runder opp. Kan kanskje fjerne? Kunne vært en kommentar?

Reply:
Tok vekk unødvendige asserts